app/Helpers/database/achievement.php

```diff
@@ -376,8 +378,12 @@ function UploadNewAchievement(
             $fields[] = "description";
         }
 
+        $recalcuateBeatTimes = false;
```

Comment:
nit, typo: should be `$recalculateBeatTimes`

Reply:
fixed